lib/queries/generate-referral-credits.js

```diff
@@ -2,8 +2,7 @@
 
 const async = require('async');
 const constants = require('../constants');
-
-const REFERRAL_CREDIT_TYPE = constants.CREDIT_TYPE.AUTO;
+const REFERRAL_CREDIT_TYPE = constants.CREDIT_TYPES.AUTO;
```

Comment:
@dylanlott @barbaraliau TODO: figure out what gives with not being able to destructure constants from service-storage-model module:
```javascript
const { CREDIT_TYPES, PROMO_CODE, STRIPE, PROMO_EXPIRES, PROMO_AMOUNT } = require('storj-service-storage-models/lib/constants');
//-- https://github.com/Storj/billing/blob/master/lib/server/routes/credits.js#L10
```